web/profiles/custom/ducore/ducore.install

```diff
@@ -119,4 +119,38 @@ function ducore_update_9002() {
   $config_path = \Drupal::service('extension.list.module')->getPath('ducore') . '/config/install/' . $config_id .'.yml';
   $data = \Symfony\Component\Yaml\Yaml::parseFile($config_path);
   \Drupal::configFactory()->getEditable($config_id)->setData($data)->save(TRUE);
-}
\ No newline at end of file
+}
+
+/**
+ * Enable CKEditor5 modules on existing installs
+ */
+function ducore_update_9003() {
+  $modules = array(
+    'ckeditor5',
+    'blazy',
+    'media',
+    'slick',
+    'token_filter'
+  );
+  \Drupal::service('module_installer')->install($modules, TRUE);
+}
+
+/**
+ * Configuration for CKEditor5 and Text Format (Rich Text) for existing installs
+ */
+function ducore_update_9004() {
+  // While Rich Text is new we made some updates to the existing file / entity browser.
+  $config_ids = array(
+    'filter.format.rich_text.yml',
+    'editor.editor.rich_text.yml',
+    'embed.button.file_browser.yml',
+    'entity_browser.browser.browse_files.yml'
+  );
+
+  // Loop through each configuration and, even if it exists, overwrite it.
+  foreach ($config_ids as $config_id) {
+    $config_path = \Drupal::service('extension.list.module')->getPath('ducore') . '/config/install/' . $config_id;
+    $data = \Symfony\Component\Yaml\Yaml::parseFile($config_path);
+    \Drupal::configFactory()->getEditable($config_id)->setData($data)->save(TRUE);
```

Comment:
Using the .yml in line #144 causes this call to the config_id to fail here.

Reply:
Thanks @kreynen-du , really good to know for both this and future config imports (which I assume will come up a few more times). I've removed those .yml extensions from the file names and hopefully that resolves it.